intelmq/bots/collectors/mail/lib.py

```diff
@@ -1,50 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
As I already mentioned in #792:

> Also, I saw that some TODOs in intelmq/bots/collectors/mail/lib.py where done, however I have not found a place where that file is actually used. I think it can be deleted.